github-process-readme.md

```diff
@@ -0,0 +1,132 @@
+# FAQ – GITHUB Process 
+
+# 1. Configure ssh keys 
+
+A) Generate new ssh keys 
+```sh
+$ ssh-keygen -o -t rsa -b 4096 -C "<seagate-email-address>"  
+```
+B) Update github settings with this public ssh key
+  - Get id_rsa.pub key (default location: /root/.ssh/id_rsa.pub) 
+
+  - Go to github ssh key setting: https://github.com/settings/keys 
+
+  - Add new ssh key and then select Enable SSO option and Authorize for this key
+
+# 2. Git configuration process 
```

Comment:
@pranav-sahasrabudhe  I earlier thought that you would link to an existing Github document.

Reply:
Yeah, after comment from @swapnil-khandare I think for now we can remove this doc as the link is already updated to doc from cortx repo. Will take the further changes in the next iteration. I am removing this doc from PR.